src/main/java/Duke.java

```diff
@@ -1,10 +1,111 @@
+                + "█░░░░░░░░░░░░░░█░░░░░░░░░░█░░░░░░░░░░░░░░█░░░░░░░░░░░░░░█░░░░░░██░░░░░░█░░░░░░██░░░░░░██░░░░░░█\n"
+                + "███████████████████████████████████████████████████████████████████████████████████████████████\n";
+        String symbol = "─────▄██▀▀▀▀▀▀▀▀▀▀▀▀▀██▄─────\n"
+                + "────███───────────────███────\n"
+                + "───███─────────────────███───\n"
+                + "──███───▄▀▀▄─────▄▀▀▄───███──\n"
+                + "─████─▄▀────▀▄─▄▀────▀▄─████─\n"
+                + "─████──▄████─────████▄──█████\n"
+                + "█████─██▓▓▓██───██▓▓▓██─█████\n"
+                + "█████─██▓█▓██───██▓█▓██─█████\n"
+                + "█████─██▓▓▓█▀─▄─▀█▓▓▓██─█████\n"
+                + "████▀──▀▀▀▀▀─▄█▄─▀▀▀▀▀──▀████\n"
+                + "███─▄▀▀▀▄────███────▄▀▀▀▄─███\n"
+                + "███──▄▀▄─█──█████──█─▄▀▄──███\n"
+                + "███─█──█─█──█████──█─█──█─███\n"
+                + "███─█─▀──█─▄█████▄─█──▀─█─███\n"
+                + "███▄─▀▀▀▀──█─▀█▀─█──▀▀▀▀─▄███\n"
+                + "████─────────────────────████\n"
+                + "─███───▀█████████████▀───████\n"
+                + "─███───────█─────█───────████\n"
+                + "─████─────█───────█─────█████\n"
+                + "───███▄──█────█────█──▄█████─\n"
+                + "─────▀█████▄▄███▄▄█████▀─────\n"
+                + "──────────█▄─────▄█──────────\n"
+                + "──────────▄█─────█▄──────────\n"
+                + "───────▄████─────████▄───────\n"
+                + "─────▄███████───███████▄─────\n"
+                + "───▄█████████████████████▄───\n"
+                + "─▄███▀───███████████───▀███▄─\n"
+                + "███▀─────███████████─────▀███\n"
+                + "▌▌▌▌▒▒───███████████───▒▒▐▐▐▐\n"
+                + "─────▒▒──███████████──▒▒─────\n"
+                + "──────▒▒─███████████─▒▒──────\n"
+                + "───────▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒───────\n"
+                + "─────────████░░█████─────────\n"
+                + "────────█████░░██████────────\n"
+                + "──────███████░░███████───────\n"
+                + "─────█████──█░░█──█████──────\n"
+                + "─────█████──████──█████──────\n"
+                + "──────████──████──████───────\n"
+                + "──────████──████──████───────\n"
+                + "──────████───██───████───────\n"
+                + "──────████───██───████───────\n"
+                + "──────████──████──████───────\n"
+                + "─██────██───████───██─────██─\n"
+                + "─██───████──████──████────██─\n"
+                + "─███████████████████████████─\n"
+                + "─██─────────████──────────██─\n"
+                + "─██─────────████──────────██─\n"
+                + "────────────████─────────────\n"
+                + "─────────────██──────────────\n";
+
+        System.out.println(logo);
+        Scanner input = new Scanner(System.in);
```

Comment:
I think since the Scanner object is also a constant, you must use the final keyword and capitalize
```suggestion
final Scanner INPUT = new Scanner(System.in);
```